### DISCUSSION TITLE
Page manually and whiteout throttling

### DISCUSSION THREAD

**Discussion by dmelendez35** [abuzuhri/Amazon-SP-API-CSharp#241] [Q&A]
Is it possible to get orders by page manually and whiteout throttling li? I don't see how to get only the first page.

Additionally, my requirement is while the first page is in waiting time because of throttling, use the time to get order items for current page orders, then continue with the next page.

There is a method "GetGetOrdersByNextToken"  but only works  if you already have a token, I don't see how to get that first token

**Comment by abuzuhri** [comment]
we have max number of pages you can use it
